contracts/src/v0.7/dev/KeeperRegistryDev.sol

```diff
@@ -664,60 +625,149 @@ contract KeeperRegistryDev is
   /**
    * @notice read the current configuration of the registry
    */
-  function getConfig()
-    external
-    view
-    override
-    returns (
-      uint32 paymentPremiumPPB,
-      uint24 blockCountPerTurn,
-      uint32 checkGasLimit,
-      uint24 stalenessSeconds,
-      uint16 gasCeilingMultiplier,
-      uint256 fallbackGasPrice,
-      uint256 fallbackLinkPrice,
-      uint32 maxPerformGas,
-      uint96 minUpkeepSpend
-    )
-  {
-    Config memory config = s_config;
+  function getConfig() external view override returns (Config memory) {
```

Comment:
`getState` or something even broader? Seems like we could condense a lot of getters in here.